app/uk/gov/hmrc/hec/services/FileCreationService.scala

```diff
@@ -37,87 +42,169 @@ trait FileCreationService {
   def createFileContent[A](
     inputType: A,
     seqNum: String,
-    partialFileName: String
+    partialFileName: String,
+    isRemaining: Boolean
   ): Either[Error, (String, String)]
 
 }
 
 @Singleton
 class FileCreationServiceImpl @Inject() (timeProvider: TimeProvider) extends FileCreationService {
 
-  val DATE_FORMATTER: DateTimeFormatter = DateTimeFormatter.ofPattern("yyyyMMdd")
-  val TIME_FORMATTER: DateTimeFormatter = DateTimeFormatter.ofPattern("HHmmss")
+  val DATE_FORMATTER: DateTimeFormatter      = DateTimeFormatter.ofPattern("yyyyMMdd")
+  val TIME_FORMATTER: DateTimeFormatter      = DateTimeFormatter.ofPattern("HHmmss")
+  val DATE_TIME_FORMATTER: DateTimeFormatter = DateTimeFormatter.ofPattern("yyyyMMdd HHmmss")
```

Comment:
I don't think there's a space needed here for date-times in the file - it says `Varchar (14)` against it which doesn't allow for the space character